django-verdant/student_profiles/forms.py

```diff
@@ -221,12 +222,16 @@ def clean_image(self):
         if self.max_size and img.size > self.max_size:
             raise forms.ValidationError(u'Please keep file size under 10MB. Current file size {}'.format(filesizeformat(img.size)))
 
+        ALLOWED_FILE_TYPES = (
+            "image/jpeg",
+            "image/gif",
+            "image/png",
+        )
+
         try:
-            dt = Image.open(img)
-            dt.load()
-            if dt.format.upper() not in ('PNG', 'JPEG', 'GIF', 'MPO'):
+            file_type = magic.from_buffer(img.read(), mime=True)
+            if not file_type in ALLOWED_FILE_TYPES:
```

Comment:
Nitpick: `if file_type not in ALLOWED_FILE_TYPES:` reads a bit more nicely (is more Pythonic), from https://www.python.org/dev/peps/pep-0008/#programming-recommendations

Reply:
Of course will change this too